README.md

```diff
@@ -24,7 +24,7 @@ Templates for [Appwrite](https://appwrite.io/) Functions. These templates can be
 | Analyze with PerspectiveAPI | [✅](node/analyze-with-perspectiveapi) | 🏗️                               | 🏗️                            | 🏗️                                        | 🏗️                          | 🏗️                             | 🏗️                            | 🏗️              | 🏗️              | 🏗️                 | 🏗️               | 🏗️                 | 🏗️                |
 | Generate PDF                | [✅](node/generate-pdf)                | 🏗️                               | 🏗️                            | 🏗️                                        | 🏗️                          | 🏗️                             | 🏗️                            | 🏗️              | 🏗️              | 🏗️                 | 🏗️               | 🏗️                 | 🏗️                |
 | Payments with Stripe        | [✅](node/payments-with-stripe)        | 🏗️                               | 🏗️                            | 🏗️                                        | 🏗️                          | 🏗️                             | 🏗️                            | 🏗️              | 🏗️              | 🏗️                 | 🏗️               | 🏗️                 | 🏗️                |
-| Push Notification with FCM  | [✅](node/push-notification-with-fcm)  | 🏗️                               | 🏗️                            | 🏗️                                        | 🏗️                          | 🏗️                             | 🏗️                            | 🏗️              | 🏗️              | 🏗️                 | 🏗️               | 🏗️                 | 🏗️                |
+| Push Notification with FCM  | [✅](node/push-notification-with-fcm)  | 🏗️                               | 🏗️                            | 🏗️                                        | 🏗️                          |                              | 🏗️                            | 🏗️              | 🏗️              | 🏗️                 | 🏗️               | 🏗️                 | 🏗️                |
```

Comment:
This is automatically generated please revert this

Reply:
I have pasted code from main branch to this. in readme. I think it revert the changes. If not please inform I will do. @loks0n